db_assessment/import_db_assessment.py

```diff
@@ -662,4 +690,103 @@ def insertErrors(invalidfiles,op_df,gcpProjectName,bq_dataset):
         print ('\nWARNING: Issues while pusing Errors into operrors table with error ', pushErr)
 
 
+def populateBT(tableName,df,dataframeornot,invalidfiles,btsource,rowsimported,importresults):
+    # Fuction to populate the importresults list which will be used to print using Beautiful Table 
+    # rowsimported of <0 is used to indicate a FAILED status 
+    # fileName='dummy'
+    tmpdataFrame=pd.DataFrame()
+
+    if btsource=='fromimportDataframeToBQ':
```

Comment:
Can we use args instead?

Reply:
args are used in latest commit for bug132 branch